WordPress/Classes/ViewRelated/NUX/LoginEpilogueTableView.swift

```diff
@@ -0,0 +1,123 @@
+import UIKit
+import WordPressShared
+
+// wrap BlogListDataSource calls to add a section for the user's info cell
+class LoginEpilogueTableView: UITableViewController {
+    var blogDataSource: BlogListDataSource
+    var blogCount: Int?
+
+    required init?(coder aDecoder: NSCoder) {
+        blogDataSource = BlogListDataSource()
+        blogDataSource.loggedin = true
+        super.init(coder: aDecoder)
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        let headerNib = UINib(nibName: "LoginEpilogueSectionHeader", bundle: nil)
+        tableView.register(headerNib, forHeaderFooterViewReuseIdentifier: "SectionHeader")
+    }
+
+    /// - Note: Copied from MeViewController which I bet @koke is happy about :P
+    fileprivate func defaultAccount() -> WPAccount? {
+        let context = ContextManager.sharedInstance().mainContext
+        let service = AccountService(managedObjectContext: context)
+        let account = service.defaultWordPressComAccount()
+        return account
+    }
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return blogDataSource.numberOfSections(in:tableView) + 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        if (section == 0) {
+            return 1
+        } else {
+            let count = blogDataSource.tableView(tableView, numberOfRowsInSection: section-1)
+            blogCount = count
+            return count
+        }
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        if (indexPath.section == 0) {
+            guard let cell = tableView.dequeueReusableCell(withIdentifier: "userInfo") as? LoginEpilogueUserInfoCell else {
+                fatalError("Failed to get a user info cell")
+            }
+            guard let account = defaultAccount() else {
+                return cell
+            }
+            if let username = account.username {
+                cell.usernameLabel?.text = "@\(username)"
+            } else {
+                cell.usernameLabel?.text = ""
+            }
+            cell.gravatarView?.downloadGravatarWithEmail(account.email, rating: .x)
+            cell.fullNameLabel?.text = account.displayName
+            return cell
+        } else {
+            let wrappedPath = IndexPath(row: indexPath.row, section: indexPath.section-1)
+            return blogDataSource.tableView(tableView, cellForRowAt: wrappedPath)
+        }
+    }
+
+    override func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        let sectionTitle: String
+        if (section == 0) {
+            sectionTitle = "LOGGED IN AS"
```

Comment:
Ah, I intended to have those localized. I'll do it in this PR—thanks for catching that